mkdocs/docs/configuration.md

```diff
@@ -222,6 +222,12 @@ catalog:
     uri: sqlite:////tmp/pyiceberg.db
 ```
 
+| Key           | Example                                                      | Description                                                                                                                                                                                                         |
+| ------------- | ------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
What do you think of adding a column for the default value:

<img width="1072" alt="image" src="https://github.com/apache/iceberg-python/assets/1134248/3dd9b143-d78d-4b88-bb75-acd3f8e8db9c">